src/component/mxgraph/MxGraphConfigurator.ts

```diff
@@ -55,5 +56,15 @@ export default class MxGraphConfigurator {
     // Disable folding for container mxCell (pool, lane, sub process, call activity) because we don't need it.
     // This also prevents requesting unavailable images (see #185) as we don't override mxGraph folding default images.
     this.graph.foldingEnabled = false;
+
+    // Pan configuration
+    if (options?.mouseNavigationSupport) {
+      this.graph.panningHandler.useLeftButtonForPanning = true;
```

Comment:
:information_source: this will be probably refactored when introducing zoom mouse wheel support

Reply:
yes